spec/features/arclight_spec.rb

```diff
@@ -84,6 +84,11 @@
           expect(page).to have_css('h3 a', text: 'Place')
           expect(page).to have_css('li .facet-label', text: 'Mindanao Island (Philippines)', visible: false)
         end
+
+        within('.blacklight-access_subjects_sim') do
+          expect(page).to have_css('h3 a', text: 'Subject')
+          expect(page).to have_css('li .facet-label', text: 'Fraternizing', visible: false)
```

Comment:
Can you add the other data in the fixture to ensure we test that long xpath query in the `access_subjects` method to catch all 4 of those elements? This is the data I see in the fixture right now:

```xml
      <subject source="mesh">Societies</subject>
      <function source="local">Fraternizing</function>
      <genreform source="aat">Photographs</genreform>
      <occupation source="mesh">Medicine</occupation>
```